zerver/tests/test_message_move_stream.py

```diff
@@ -1829,3 +1829,29 @@ def test_move_message_to_private_hidden_history_with_new_member(self) -> None:
             ),
             True,
         )
+
+    def test_move_message_update_stream_active_status(self) -> None:
+        (user_profile, old_stream, new_stream, msg_id, msg_id_later) = self.prepare_move_topics(
+            "iago", "test move stream", "new stream", "test"
+        )
+
+        # Delete all messages in new stream and mark it as inactive.
+        Message.objects.filter(recipient__type_id=new_stream.id, realm=user_profile.realm).delete()
+
+        new_stream.is_recently_active = False
+        new_stream.save()
```

Comment:
Might be better if these two lines were to call `check_update_all_streams_active_status`

Reply:
Yeah, done.